mmdeploy/apis/ncnn/__init__.py

```diff
@@ -1,11 +1,11 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 from mmdeploy.backend.ncnn import from_onnx as _from_onnx
-from mmdeploy.backend.ncnn import is_available, is_plugin_available
+from mmdeploy.backend.ncnn import is_available, is_customops_available
 from ..core import PIPELINE_MANAGER
 
 from_onnx = PIPELINE_MANAGER.register_pipeline()(_from_onnx)
 
-__all__ = ['is_available', 'is_plugin_available', 'from_onnx']
+__all__ = ['is_available', 'is_customops_available', 'from_onnx']
```

Comment:
not sure if `is_customops_available` is appropriate.

Reply:
Not sure too.